### DISCUSSION TITLE
I ❤️ Recoil

### DISCUSSION THREAD

**Discussion by mrlubos** [facebookexperimental/Recoil#1581] [General]
That's it. I've been using Recoil at least since version `0.0.10` and it's been fun following the development and updates. The library is very light-weight, does its job well, and gets out of the way. Excited to keep following the future progress!

**Comment by drarmstr** [comment]
Thank you for the kind words of support!

**Comment by baptistefkt** [comment]
> That's it. I've been using Recoil at least since version `0.0.10` and it's been fun following the development and updates. The library is very light-weight, does its job well, and gets out of the way. Excited to keep following the future progress!

Hi, what do you mean by "very light-weight"? I'm curious, because in term of bundle size, it looks like one of the heaviest out there: 

- recoil: 22.3kB
- redux + react-redux: 7kB
- jotai: 3.2kB
- valtio: 2.6kB
- zustand: <1kB

(minified + gzipped sizes)

**Reply by mrlubos** [reply]
Hey! Thanks for pointing that out. What I had in mind by light-weight is that it's very easy to implement into the codebase. When I used to work with Redux, it was so much boilerplate code to integrate a simple state - reducers, actions, HOCs, and it was nowhere near as fast as Recoil. Would love to hear about your experience with the other libraries you mentioned!

**Reply by mrlubos** [reply]
And while we're on the topic, @drarmstr  are there any plans to reduce the bundle size?

**Comment by ayoubkhan558** [comment]
I don't love I find it hard to use.
